payments-core/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -223,11 +223,8 @@ constructor(
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     fun isLpmLevelSetupFutureUsageSet(code: PaymentMethodCode): Boolean {
         return isTopLevelSetupFutureUsageSet() || paymentMethodOptionsJsonString?.let {
-            JSONObject(paymentMethodOptionsJsonString)
-                .optJSONObject(code)
-                ?.optString("setup_future_usage")?.let {
-                    true
-                } ?: false
```

Comment:
The issue here is that `jsonObject.optString(key)` returns an empty string if `key` isn’t present, meaning that this code snippet would return true if `payment_method_options[code]` exists, even if `payment_method_options[code]["setup_future_usage"]` doesn’t.